packages/schemas/src/gen/utils.ts

```diff
@@ -78,6 +78,29 @@ export const findFirstParentheses = (value: string): Optional<ParenthesesMatch>
   return matched ? rest : undefined;
 };
 
+// Split at each comma that is not in parentheses
+export const splitAtCommasOutsideParentheses = (value: string) =>
+  Object.values(value).reduce<{ result: string[]; count: number }>(
+    (previous, current) => {
+      const { result } = previous;
```

Comment:
If you want to deconstruct `result` from `previous`, then you should also deconstruct `count` from it